### ISSUE TITLE
为什么我推理输出的结果是一行感叹号？

### ISSUE THREAD

**Issue by xiao1122331998** [XiaomiMiMo/MiMo-VL#8]
代码：
import torch
from transformers import Qwen2_5_VLForConditionalGeneration, AutoTokenizer, AutoProcessor
from qwen_vl_utils import process_vision_info

model = Qwen2_5_VLForConditionalGeneration.from_pretrained(
    "/360/dongwoAIData/MiMo-VL-7B-SFT", torch_dtype=torch.float16, device_map="cuda:5"
)
min_pixels = 256*28*28
max_pixels = 1280*28*28
processor = AutoProcessor.from_pretrained("/360/dongwoAIData/MiMo-VL-7B-SFT", min_pixels=min_pixels, max_pixels=max_pixels)

messages = [
    {
        "role": "user",
        "content": [
            {
                "type": "image",
                "image": "/mnt/xiaohaoliang/Qwen2.5/1.JPG",
            },
            {"type": "text", "text":"OCR图中文字。"},
        ],
    }
]
# Preparation for inference
text = processor.apply_chat_template(
    messages, tokenize=False, add_generation_prompt=True
)

image_inputs, video_inputs = process_vision_info(messages)
inputs = processor(
    text=[text],
    images=image_inputs,
    videos=video_inputs,
    padding=True,
    return_tensors="pt",
)
inputs = inputs.to("cuda:5")
torch.cuda.empty_cache()
# Inference: Generation of the output
generated_ids = model.generate(**inputs, max_new_tokens=128)
generated_ids_trimmed = [
    out_ids[len(in_ids):] for in_ids, out_ids in zip(inputs.input_ids, generated_ids)
]
output_text = processor.batch_decode(
    generated_ids_trimmed, skip_special_tokens=True, clean_up_tokenization_spaces=False
)
print(output_text)

输出结果：
Loading checkpoint shards: 100%|███████████████████████████████████████████████████████████████████████████████████| 4/4 [00:11<00:00,  2.84s/it]
Using a slow image processor as `use_fast` is unset and a slow processor was saved with this model. `use_fast=True` will be the default behavior 
in v4.52, even if the model was saved with a slow processor. This will result in minor differences in outputs. You'll still be able to use a slow
 processor with `use_fast=False`.
You have video processor config saved in `preprocessor.json` file which is deprecated. Video processor configs should be saved in their own `vide
o_preprocessor.json` file. You can rename the file or load and save the processor back which renames it automatically. Loading from `preprocessor
.json` will be removed in v5.0.
['!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!']

**Comment by mediawall** [comment]
没有早点看到这个消息，花了我一天时间，哎

**Comment by advanture917** [comment]
我也遇到了这种情况，` torch_dtype=torch.float16` ,加载模型为fp16会导致这种情况，改为`torch_dtype=torch.float16` 就可以了。或者可以尝试在加载模型权重格式fp16的情况下，对输入图片resize（我测试的200*200以内的都可以正常输出）

![Image](https://github.com/user-attachments/assets/6fc31035-80fb-4cb8-81af-4d3027c4bc0c)

**Comment by advanture917** [comment]
![Image](https://github.com/user-attachments/assets/733c80f2-cff2-4883-8535-3b1897260751)
在使用 FP16 精度推理时，我发现当输入图像尺寸较大时，模型的 ViT 阶段会产生 NaN 值，进而导致后续生成结果异常（如输出全是!!!!...或是bits bits...）。

**Comment by xiao1122331998** [comment]
@advanture917 非常感谢您，我的问题解决了
